src/client/page.ts

```diff
@@ -220,7 +220,7 @@ export class Page extends ChannelOwner<channels.PageChannel, channels.PageInitia
     return this._browserContext;
   }
 
-  async opener(): Promise<Page | null> {
+  opener(): Page | null {
```

Comment:
This is not fully backwards compatible as it will break code like this:
```js
  page.opener().then(p => console.log(p.url()));
```
I'm also not quite sure about implications of removing `async` from a function in python. Considering that I'd leave current signature as is, the implementation can still be synchronous.

Reply:
Yeah. This is more to open the conversation and how we want to proceed with a method that we do know it's async.
Maybe starting with a custom sync csharp method?

Reply:
I'd keep it async for consistency with other languages, the implementation can be sync or async the client should not care. Does it cause any issues with the implementation?

Reply:
Nope, we can return a resolved task.